From 60b8b1379f2eb154a3c58c00dabbf0f9f0bd5445 Mon Sep 17 00:00:00 2001
From: Roger Zanoni <rzanoni@igalia.com>
Date: Thu, 4 May 2023 10:42:06 +0200
Subject: [PATCH 4/8] Don't use DRI for renesas

This adds the use_dri flag that defauls to true and
can be disabled if we are building CEF using ozone + wayland

Signed-off-by: Roger Zanoni <rzanoni@igalia.com>
---
 content/gpu/BUILD.gn | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/content/gpu/BUILD.gn b/content/gpu/BUILD.gn
index 9f5f303b2f220..f7639030ac30c 100644
--- a/content/gpu/BUILD.gn
+++ b/content/gpu/BUILD.gn
@@ -8,6 +8,10 @@ import("//build/config/ui.gni")
 import("//gpu/vulkan/features.gni")
 import("//media/media_options.gni")
 
+declare_args() {
+  use_dri = true
+}
+
 # See //content/BUILD.gn for how this works.
 group("gpu") {
   visibility = [ "//content/*" ]  # This is an internal content API.
@@ -137,8 +141,8 @@ target(link_target_type, "gpu_sources") {
   }
 
   # Use DRI on desktop Linux builds.
-  if (current_cpu != "s390x" && current_cpu != "ppc64" && is_linux &&
-      !is_castos) {
+  if (use_dri && current_cpu != "s390x" && current_cpu != "ppc64" &&
+      is_linux && !is_castos) {
     configs += [ "//build/config/linux/dri" ]
   }
 }
-- 
2.37.2

